scripts/add_confirmed_to_existing_users_token.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Migration to add a confirmed boolean to User.email_verifications
+"""
+import sys
+import logging
+from modularodm import Q
+
+from website.app import init_app
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from framework.auth import User
+
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration():
+    for user in User.find(Q('email_verifications', 'ne', {})):
+        for token in user.email_verifications:
+            if 'confirmed' not in token:
+                user.email_verifications[token]['confirmed'] = False
```

Comment:
`.get('confirmed', False)` should work in places where you need to access it but it might not be defined. However, I just wanted to confirm the semantics behind this flag -- if it's significantly more difficult or ugly to not do this migration, it's fine the way it is.